### ISSUE TITLE
Package.getStylesheetsPath is deprecated.

### ISSUE THREAD

**Issue by kaisellgren** [Nicolab/atom-local-history#22]
Store package style sheets in the `styles/` directory instead of `stylesheets/` in the `local-history` package
```
Package.getStylesheetsPath (C:\Users\kai\AppData\Local\atom\app-0.190.0\resources\app\src\package.js:433:9)
Package.getStylesheetPaths (C:\Users\kai\AppData\Local\atom\app-0.190.0\resources\app\src\package.js:444:32)
```

**Comment by Nicolab** [comment]
ok I'll look that, thank you

**Comment by Nicolab** [comment]
Released to local-history@v3.0.0 (compatible Atom v0.192 and higher).

Thanks :)
